README.md

```diff
@@ -42,7 +42,7 @@ async def main():
     await nsq.subscribe('test_topic', 'channel1', 2)
     processed_messages = 0
     async for message in nsq.messages():
-        print('Message #{}: {}'.format(processed_messages, message.body))
+        print('Message #{}: {}'.format(processed_messages, str(message)))
```

Comment:
```suggestion
        print('Message #{}: {}'.format(processed_messages, message))
```

There is no need to convert explicitly message to str if `str.format(...)` using.